install.sh

```diff
@@ -183,6 +182,14 @@ User=root
 WantedBy=multi-user.target
 EOF
 
+    sudo mkdir -p /var/run/flannel
+    sudo mkdir -p /var/run/kubelet
+    sudo mkdir -p /var/lib/kubelet/pods
+    sudo mkdir -p /var/run/secrets/kubernetes.io/serviceaccount
+    sudo mkdir -p /var/hpvolumes
+    sudo semodule -i /tmp/microshift.pp
```

Comment:
will /tmp survive?

Reply:
Once the pp file is loaded it is no longer needed. We could potentially add in a delete of the file once loaded

Reply:
as long as it is persisted, it is good